ironfish/src/strategy.test.slow.ts

```diff
@@ -213,7 +213,7 @@ describe('Demonstrate the Sapling API', () => {
         workerPool: new WorkerPool(),
         consensus: new TestnetConsensus(consensusParameters),
       })
-      const minersFee = await strategy.createMinersFee(BigInt(0), 0, key.spendingKey)
+      const minersFee = await strategy.createMinersFee(0, 0, key.spendingKey)
```

Comment:
```suggestion
      const minersFee = await strategy.createMinersFee(0n, 0, key.spendingKey)
```

Missing `n` to make this a bigint literal